specification/devcenter/DevCenter/DevBox/parameters.tsp

```diff
@@ -0,0 +1,37 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "../shared/models.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+namespace DevCenterService;
+
+alias SnapshotIdQueryParameter = {
```

Comment:
please use `model XXParameter` instead of alias. alias will generate anonymous models instead of a shared `XXParameter` in the parameter section.